relay/nakama/allowlist/allowlist.go

```diff
@@ -3,54 +3,33 @@ package allowlist
 import (
 	"context"
 	"crypto/rand"
-	"database/sql"
 	"encoding/hex"
 	"encoding/json"
+	"errors"
 	"fmt"
-	"os"
-	nakamaerrors "pkg.world.dev/world-engine/relay/nakama/errors"
+	"github.com/heroiclabs/nakama-common/runtime"
+	"github.com/rotisserie/eris"
 	"pkg.world.dev/world-engine/relay/nakama/signer"
 	"pkg.world.dev/world-engine/relay/nakama/utils"
-	"strconv"
 	"strings"
-
-	"github.com/heroiclabs/nakama-common/runtime"
-	"github.com/rotisserie/eris"
 )
 
 var (
-	allowlistEnabledEnvVar = "ENABLE_ALLOWLIST"
-	allowlistEnabled       = false
-	allowlistKeyCollection = "allowlist_keys_collection"
-	allowedUsers           = "allowed_users"
+	EnabledEnvVar = "ENABLE_ALLOWLIST"
+	Enabled       = false
+	KeyCollection = "allowlist_keys_collection"
+	AllowedUsers  = "allowed_users"
```

Comment:
Is there any particular reason why these need to be public variables?

Reply:
They are used in `rpc.go` and `rpc_test.go`, and the last two are used in `allowlist.go` as well